Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: help install-coding-standard-dependencies coding-standard-fix coding-standard-check
+.DEFAULT_GOAL := help
+
+help:                                                                           ## shows this help
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_\-\.]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+install-coding-standard-dependencies:                ## install dependencies for coding-standard checks tooling
+	cd tools/php-cs-fixer && composer update
+
+coding-standard-fix:                                 ## apply automated coding standard fixes
+	./tools/php-cs-fixer/vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php
+
+coding-standard-check:                               ## check coding-standard compliance
+	./tools/php-cs-fixer/vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php --dry-run
```

Comment:
Are we using this anywhere? It's a nice shortcut, but I'd rather just rely on CI to run this stuff.

Reply:
Only locally. GitHub actions runs a reusable workflow.

I don't think these are a good idea. 

Developers should be able to run exactly what runs in CI, which they can't here.

Reply:
Happy to remove the makefile.
The issue of the shared workflow remains.